src/components/Header/HeaderAccountBalance.jsx

```diff
@@ -1,88 +1,135 @@
-import {
-  Grid,
-  Container,
-  Divider,
-  Stack,
-  Typography,
-  ListItemIcon,
-} from '@mui/material';
+import { Container, Box, Divider, Stack, Typography } from '@mui/material';
 import { HistoryOutlined, CalendarTodayOutlined } from '@mui/icons-material';
-import { flexbox } from '@mui/system';
 
-const accountWidth = 240;
+const AccountBalance = (props) => {
+  const filterAccountBalance = () => {
+    return props.data.reduce((prev, current) => {
+      return current.status === 'ISSUED' || current.status === 'OVERDUE'
+        ? prev + current.amount
+        : prev;
+    }, 0);
+  };
+
+  const invoiceOverdue = () => {
+    return props.data.reduce((prev, current) => {
+      return current.status === 'OVERDUE' ? prev + current.amount : prev;
+    }, 0);
+  };
+
+  const calculateDate = () => {
+    return props.data.filter((element) => {
+      return element.status.includes('ISSUED') ? element : '';
+    })[0];
+  };
+  const getInvoice = calculateDate().due.split('/');
+  const newNextInvoice =
+    getInvoice[0] + '/' + (Number(getInvoice[1]) + 1) + '/' + getInvoice[2];
```

Comment:
```suggestion
  const nextIssueDate =
    latestInvoiceDueDate[0] + '/' + (Number(latestInvoiceDueDate[1]) + 1) + '/' + latestInvoiceDueDate[2];
```

This applies to your other usecase, in the `InvoiceDetails` component. You can even extract this formatting date logic into a function in a separate file, and use it in both places.